src/js/apps/patients/worklist/worklist-owned-by_app.js

```diff
@@ -0,0 +1,172 @@
+import moment from 'moment';
+import store from 'store';
+
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+import { dateSort } from 'js/utils/sorting';
+import intl from 'js/i18n';
+
+import App from 'js/base/app';
+
+import FiltersApp from './filters_app';
+
+import { ListView, LayoutView, SortDropList } from 'js/views/patients/worklist/worklist_views';
+
+const i18n = intl.patients.worklist.worklistApp;
+
+const sortOptions = new Backbone.Collection([
```

Comment:
ya know.. 100% of this is view related.    Why don't we move `sortOptions` to `worklist_views` for now.